Sources/protoc-gen-swiftgrpc/options.swift

```diff
@@ -85,7 +85,7 @@ class GeneratorOptions {
   }
 
   static func parseParameter(string: String?) -> [(key: String, value: String)] {
-    guard let string = string, string.characters.count > 0 else {
+    guard let string = string, string.isEmpty else {
```

Comment:
This isEmpty condition should be inverted, i.e. it should be !string.isEmpty instead of string.isEmpty. Fortunately, our plugin test (`make test-plugin`) caught it!

If you don't mind, please update that in your PR. When the tests pass, I'll merge it.

Thank you!